diff --git a/node_modules/react-native-select-dropdown/src/styles.js b/node_modules/react-native-select-dropdown/src/styles.js
index d4d0d61..a872f18 100644
--- a/node_modules/react-native-select-dropdown/src/styles.js
+++ b/node_modules/react-native-select-dropdown/src/styles.js
@@ -1,5 +1,5 @@
-import {StyleSheet, Dimensions, I18nManager} from 'react-native';
-const {width, height} = Dimensions.get('window');
+import { StyleSheet, Dimensions, I18nManager } from 'react-native';
+const { width, height } = Dimensions.get('window');
 
 const styles = StyleSheet.create({
   dropdownButton: {
@@ -65,7 +65,7 @@ const styles = StyleSheet.create({
   //////////////////////////////////////
   shadow: {
     shadowColor: '#000',
-    shadowOffset: {width: 0, height: 6},
+    shadowOffset: { width: 0, height: 6 },
     shadowOpacity: 0.1,
     shadowRadius: 10,
     elevation: 10,
